packages/now-static-build/src/frameworks.ts

```diff
@@ -413,7 +413,8 @@ const frameworkList: Framework[] = [
   {
     name: 'Jekyll',
     slug: 'jekyll',
-    buildCommand: 'jekyll build',
+    buildCommand: 'jekyll build && bundle clean',
```

Comment:
I don't think `bundle clean` should be here either. Its not part of the build command.

Reply:
Oh, I see. I was just trying to do some sort of cleanup afterwards to prevent the cache from growing indefinitely whenever a gem is updated.

I'll remove it now, then.